tests/node-18.0/tests.js

```diff
@@ -1,37 +1,57 @@
 const fetch = require("node-fetch");
 
-/*
-    'req' variable has:
-        'headers' - object with request headers
-        'payload' - object with request body data
-        'variables' - object with function variables
-    'res' variable has:
-        'send(text, status)' - function to return text response. Status code defaults to 200
-        'json(obj, status)' - function to return JSON response. Status code defaults to 200
-    
-    If an error is thrown, a response with code 500 will be returned.
-*/
+module.exports = async (context) => {
+    const action = context.req.headers['x-action'];
 
-module.exports = async (req, res) => {
-    const payload = JSON.parse(req.payload === '' ? '{}' : req.payload);
+    switch (action) {
+        case 'plaintextResponse':
+            return context.res.send('Hello World 👋');
+        case 'jsonResponse':
+            return context.res.json({ json: true, message: 'Developers are awesome.' });
+        case 'fileResponse':
+            return context.res.file(__dirname + '/resources/picture.png', undefined, { 'content-type': 'image/png' });
```

Comment:
Discuss. Ideally no need for `__dirname`, ideally uses `_resources` and not the copy